authors/apps/articles/tests/base_test.py

```diff
@@ -50,3 +51,15 @@ def create_article(self):
 
     def single_article_url(self, slug):
         return reverse('articles:article-details', args=[slug])
+
+    def like_article(self, slug):
+        url = self.client.put(
+            reverse('articles:likes', kwargs={"slug": slug}),
+            format='json')
+        return url
+
+    def dislike_article(self, slug):
+        url = self.client.put(
+            reverse('articles:dislikes', kwargs={"slug": slug}),
```

Comment:
same as above ensure the variable name represents what it's assigned. E.g here the `url` = `reverse('articles:dislikes', kwargs={"slug": slug})` not the request by test client as labelled.

Reply:
done